### DISCUSSION TITLE
Unexpected token '??=' when running a new project created with cargo create-tauri-app --beta

### DISCUSSION THREAD

**Discussion by MarcMouries** [tauri-apps/tauri#10337] [Q&A]
I just created a new project created with cargo create-tauri-app --beta

I used the following options

✔ Choose which language to use for your frontend · **TypeScript / JavaScript - (pnpm, yarn, npm, bun)**
✔ Choose your package manager · **bun**
✔ Choose your UI template · **Svelte - (https://svelte.dev/)**
✔ Choose your UI flavor · **JavaScript**
✔ Would you like to setup the project for mobile as well? · **yes**

And I am encountering an error right after i try to run the project that was generated.

```
$bun run tauri dev
$ tauri dev
    Running BeforeDevCommand (`bun run dev`)
$ vite dev
(node:83330) UnhandledPromiseRejectionWarning: SyntaxError: Unexpected token '??='
    at Loader.moduleStrategy (internal/modules/esm/translators.js:149:18)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:83330) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:83330) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    Warn Waiting for your frontend dev server to start on http://localhost:1420/...
    Warn Waiting for your frontend dev server to start on http://localhost:1420/...
```

**Comment by FabianLars** [comment]
Which `bun` version are you using?

**Reply by MarcMouries** [reply]
I am using Bun v1.1.20.
I tried again today and limited the options. I went with Bun and Vanilla and no Mobile and it worked this time. 
Yesterday i had chosen Svelte and Mobile

**Reply by FabianLars** [reply]
Hmm, what about your nodejs version? `??=` should be supported since 15.0 according to MDN.
